Annotato/Annotato/Adapters/AnnotatoHTTPService.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+protocol AnnotatoHTTPService {
+    func get(url: String, params: [String: String]) async throws -> Data
+
+    func get(url: String) async throws -> Data
+
+    func post(url: String, data: Data) async throws -> Data
+
+    func put(url: String, data: Data) async throws -> Data
+
+    func delete(url: String) async throws -> Data
```

Comment:
```suggestion
    func get(url: String, params: [String: String]) async throws -> Data
    func get(url: String) async throws -> Data
    func post(url: String, data: Data) async throws -> Data
    func put(url: String, data: Data) async throws -> Data
    func delete(url: String) async throws -> Data
```

I think remove linespaces to standardise with the other XXService.